openedx/edx-enterprise-data
# ENT-2033: Added ability to include/exclude date from report configuration file name

Added "Include Date" field in Enterprise Customer Report Configuration Admin Form as mentioned in  ENT-2033. 

Steps to check it:
1. Go to LMS and add Enterprise Customer
2. Go to Enterprise Customer -> Enterprise customer reporting configurations
2. Add Enterprise Customer Reporting Configuration
3. You can see that the form now shows the "Include Date" field. 
4. Run LMS_OAUTH_HOST='http://localhost:18000' LMS_ROOT_URL='http://localhost:18000' LMS_FERNET_KEY='DUMMY KEY CHANGE BEFORE GOING TO PRODUCTION' PYTHONPATH=/../edx-enterprise-data python enterprise_reporting/send_enterprise_reports.py -e [EnterpriseCustomerUUID]


Test Cases: test_reporter.py, test_utils.py::TestCompressEncrypt
